src/functions/process-stream-event.ts

```diff
@@ -44,7 +44,7 @@ export const processStreamEvent: Handler = async (
 
     for await (const record of event.Records) {
       const id = record.messageId;
-      const dynamoRecord: DynamoDBRecord = JSON.parse(JSON.parse(record.body).Message) as DynamoDBRecord;
+      const dynamoRecord: DynamoDBRecord = JSON.parse(record.body) as DynamoDBRecord;
```

Comment:
not quite, record.body is the whole SNS event, needs to be on the parsed event dynamodb.NewImage and may also need unmarshalling, bet less sure about that bit, need to evaluate what it was doing pre changes.

Reply:
ignore me I am wrong